fish_command_timer.fish

```diff
@@ -123,7 +123,7 @@ else
   set -e fish_command_timer_enabled
 end
 
-set fish_command_timer_start_time
+set fish_command_timer_start_time (fish_command_timer_get_ts)
```

Comment:
except that if I don't do this, the variable is empty, and when the shell tries to execute the line below

`set -l command_time (math "$command_end_time - $command_start_time")`

the variable isn't set, so math is unhappy and errors are displayed on initial source